_rules/SVG-image-has-name.md

```diff
@@ -0,0 +1,141 @@
+---
+name: SVG image has accessible name
+description: |
+  This rule checks that each SVG image element that is included in the accessibility tree, has an accessible name.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any [SVG](https://www.w3.org/2000/svg) element with the [semantic role](#semantic-role) of `img` or `graphics-document`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
This rule is confusing about what it applies to. If this applies to all elements with the `svg` namespace, you need to add different test cases. If it only applies to elements with the `svg` tagname, you should change the applicability.

Reply:
Have updated the applicability and added some test cases but not sure if I have addressed your concerns - can you give this another review @WilcoFiers ?